schema/Wheels-01.yaml

```diff
@@ -0,0 +1,14 @@
+$id: http://open.qa/api/schema/Wheels-01.yaml
+$schema: http://json-schema.org/draft-04/schema#
+description: Spec for loadable modules
```

Comment:
modules? Shouldn't that be "wheels" or "plugins" to not introduce yet another word for the same?